subsys/bluetooth/host/conn.c

```diff
@@ -1085,6 +1085,16 @@ int bt_conn_set_security(struct bt_conn *conn, bt_security_t sec)
 
 	return err;
 }
+
+bt_security_t bt_conn_get_security(struct bt_conn *conn)
+{
+	return conn->sec_level;
+}
+#else
+bt_security_t bt_conn_get_security(struct bt_conn *conn)
+{
+	return BT_SECURITY_L1;
```

Comment:
Should it be L0 for BR/EDR? (It looks like conn->sec_level is not initialized for BR/EDR connections, see `bt_conn_add_br`)

Reply:
Well L0 is only used for SDP iirc.